examples/python/model-qa.py

```diff
@@ -99,7 +99,7 @@ def main(args):
         if args.verbose: print("Generator created")
 
         # Append system and input tokens to the generator
-        generator.append_tokens(system_tokens + input_tokens)
+        generator.append_tokens(list(system_tokens) + list(input_tokens))
```

Comment:
why not `np.concatenate([system_tokens, input_tokens])`?

Reply:
yep, makes more sense.